src/pages/ViewEmailPage/index.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react"
+import {useNavigate, useParams } from "react-router-dom"
+
+function ViewEmailPage() { 
+    const {id} = useParams()
+    const navigate = useNavigate()
+    
+    const [name, setName] = useState ("")
+    const [emailAddress, setEmailAddress] = useState("")
+    const [subject, setSubject] = useState("")
+    const [body, setBody] = useState("")
+    const [date, setDate] = useState("")
+    const [isDeleted, setIsDeleted] = useState()
+
+    function getEmailData() {
+        fetch(`https://email-client-api.dev.io-academy.uk/emails/${id}`)
+            .then(res => res.json())
+            .then(emailData => {
+                setName(emailData.data.email.name)
+                setEmailAddress(emailData.data.email.email)
+                setSubject(emailData.data.email.subject)
+                setBody(emailData.data.email.body)
+                setDate(emailData.data.email.date_created)     
+                setIsDeleted(emailData.data.email.deleted)           
+            })
+    }
+
+    function deleteEmail() {
+       setIsDeleted(1)
+            fetch(`https://email-client-api.dev.io-academy.uk/emails/${id}`, {
+                method: 'DELETE',  
+                headers: {
+                    "content-type": "application/json"
+                },
+                
+                body:JSON.stringify({
+                deleted:isDeleted
```

Comment:
No body is needed at all for this one, just sending a DELETE request to the API with the id in the url (as you already have) is enough